src/test/scala/mockws/Helpers.scala

```diff
@@ -0,0 +1,13 @@
+package mockws
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import play.api.mvc.{DefaultActionBuilder, PlayBodyParsers}
+import scala.concurrent.ExecutionContext.Implicits._
+
+object Helpers {
+  implicit val as = ActorSystem()
+  implicit val materializer = ActorMaterializer()
```

Comment:
Registering a shutdown hook does the job, since it will create the actor system and shut it down on every `sbt ~test` code change. So it won't cause an infinite number of actor systems to appear.